.bazelrc

```diff
@@ -2,3 +2,6 @@
 build --workspace_status_command hack/build/print-workspace-status.sh
 # Show timestamps with each bazel message
 build --show_timestamps
+
+# import per-user options
+try-import %workspace%/user.bazelrc
```

Comment:
The other alternative is to set this at $HOME/.bazelrc - though that will apply to all bazel projects, not just this one 😊

Reply:
Yeah, want to make to limit these configs!